apps/openmw/mwscript/miscextensions.cpp

```diff
@@ -941,13 +937,11 @@ namespace MWScript
                         case 'f':
                             output << ptr.getCellRef().getRefId() << "." << var << ": " << ptr.getRefData().getLocals().getFloatVar(script, var);
                             break;
-                        default:
-                            output << "unknown local '" << var << "' for '" << ptr.getCellRef().getRefId() << "'";
-                            break;
+                        // Do nothing otherwise
```

Comment:
Useless comment. It's obvious that there is nothing else in this switch statement.

Reply:
Arguably it demonstrates to the reader that all possible switch cases were considered. Ok, I'll remove it.